new_frontend/src/pages/bonafide-certificate/index.tsx

```
@@ -1,8 +1,95 @@
+            <ProfileWrapper
+              name={student.name}
+              classSection={student.classSection}
+              student_name={student.student_name}
+              image={student.image}
+              reference_number={student.reference_number}
+              first_name={student.first_name}
+              custom_division={student.custom_division}
+              last_name={student.last_name}
+              program_name={student.program_name}
+              isLoading={false}
+              children={<BonafideChildComponent student={student} />}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)